diff -Naupr vobcopy-1.1.1.orig/vobcopy.c vobcopy-1.1.1/vobcopy.c
--- vobcopy-1.1.1.orig/vobcopy.c	2008-02-19 06:27:04.000000000 +0100
+++ vobcopy-1.1.1/vobcopy.c	2008-05-13 00:19:18.000000000 +0200
@@ -519,7 +519,7 @@ and potentially fatal."  - Thanks Leigh!
       strcpy( tmp_path, pwd );
       strcat( tmp_path, "vobcopy.bla" );
       fprintf( stderr, "[Hint] Quiet mode - All messages will now end up in %s\n", tmp_path );
-      if ( ( temp = open( tmp_path , O_CREAT | O_EXCL ) ) == -1 )
+      if ( ( temp = open( tmp_path , O_CREAT | O_EXCL , 0644 ) ) == -1 )
 	{
 	  printf( "[Error] Error: %s\n", strerror( errno ) );
 	  if ( errno == EEXIST )
@@ -575,7 +575,7 @@ and potentially fatal."  - Thanks Leigh!
       strcat( logfile_name, VERSION );
       strcat( logfile_name, ".log" );
       strcat( logfile_path, logfile_name );
-      if ( ( temp = open ( logfile_path , O_CREAT | O_EXCL ) ) == -1 )
+      if ( ( temp = open ( logfile_path , O_CREAT | O_EXCL , 0644 ) ) == -1 )
 	{
 	  printf( "[Error] Error: %s\n", strerror( errno ) );
 	  if ( errno == EEXIST )
